.github/workflows/conformance-aws-cni-v1.10.yaml

```diff
@@ -66,8 +66,8 @@ env:
   cilium_base_version: "1.10"
   cilium_cli_version: v0.10.7
   check_url: https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}
-  eksctl_version: v0.94.0
-  kubectl_version: v1.23.6
+  eksctl_version: v0.100.0
+  kubectl_version: v1.24.0
```

Comment:
might be related to helm version as well 

https://github.com/cilium/cilium-cli/pull/895

Reply:
yep, it is. I was trying to see if bumping these would fix up kubeconfig so that newer helm version would work, but I guess would be easier to fix the helm version...

Reply:
you are right, by the time I sent above PR, eksctl was still with 100.rc0 version so i decided to go with helm.